src/drivers/bitbucket_cloud.js

```diff
@@ -254,10 +258,13 @@ class BitbucketCloud {
 
     if (response.status > 300) {
       try {
-        const {
-          error: { message }
-        } = await response.json();
-        throw new Error(message);
+        const json = await response.json();
+        winston.debug(json);
+        // Attempt to get additional context. We have observed two different error schemas
+        // from BitBucket API responses: `{"error": {"message": "Error message"}}` and
+        // `{"error": "Error message"}`. Fall back to the empty string to ensure we don't
+        // print `undefined` to the user.
+        throw new Error(json.error.message || json.error || '');
```

Comment:
```suggestion
        throw new Error(json.error.message || json.error);
```

It looks like `new Error(undefined).message === ""` and there would be no need of providing an empty string fallback.

Reply:
Ah you're right!

Retraced my steps to see how I managed to get `undefined` in the error message. Seems like I was running `0.9.0` instead of `master` when I got this output. So that bit was already fixed by https://github.com/iterative/cml/commit/44806c0d583fe359fdf34389332d802975f6f501 :smile: